CMakeLists.txt

```diff
@@ -81,6 +81,8 @@ function(determine_version OUTPUT_VARIABLE)
 
   if(GIT_RELEASE)
     set(NEW_VERSION ${GIT_RELEASE})
+  elseif(GIT_VERSION_MATCH AND MULTIPASS_BUILD_ID)
+    set(NEW_VERSION ${GIT_TAG}.${CMAKE_MATCH_1}.${MULTIPASS_BUILD_ID}+${CMAKE_MATCH_2})
```

Comment:
ATM this is the first piece of info I look for to quickly identify the package. A date would not be so useful, unless it included the time (I often build multiple versions the same day). And the hash is just not memorable.

Reply:
Yeah it'd have to be like `0.9.0-dev.201910171808+g20cca791` - we could maybe drop the century.

Reply:
Personally I prefer the other option. One just needs to know that number-of-commits-since-tag does not translate linearly to recency (because of branches). And the date/time should already be a property of the file, no?

Reply:
OK I think I'm ok with the commit count after all…